### PR TITLE
Setup child in care backend

### PR DESCRIPTION
**Pull request by Cajan19**
Add model, db, service, controller and test

### CODE REVIEW

File: backend-nannyway/src/main/java/de/neuefische/nannyway/controller/ChildInCareController.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+import de.neuefische.nannyway.model.AddChildInCareDto;
+import de.neuefische.nannyway.model.ChildInCare;
+import de.neuefische.nannyway.service.ChildInCareService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@RequestMapping("/api/tageskinder")
```
Comment (fabianschmauder) [inline backend-nannyway/src/main/java/de/neuefische/nannyway/controller/ChildInCareController.java:11]: maybe in english?

File: backend-nannyway/src/main/java/de/neuefische/nannyway/model/AddChildInCareDto.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class AddChildInCareDto {
+
+    private String firstName;
```
Comment (fabianschmauder) [inline backend-nannyway/src/main/java/de/neuefische/nannyway/model/AddChildInCareDto.java:14]: you can use annotations to verify that the name ist not empty

Comment (fabianschmauder) [reply backend-nannyway/src/main/java/de/neuefische/nannyway/model/AddChildInCareDto.java:14]: than you also have to add @Valid to the controller

File: backend-nannyway/src/test/java/de/neuefische/nannyway/controller/ChildInCareControllerTest.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+import org.springframework.http.*;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class ChildInCareControllerTest {
```
Comment (fabianschmauder) [inline backend-nannyway/src/test/java/de/neuefische/nannyway/controller/ChildInCareControllerTest.java:27]: i would also test that you get forbidden when you are not logged in

File: backend-nannyway/src/main/java/de/neuefische/nannyway/controller/ChildInCareController.java (hunk 1 above)
Comment (Cajan19) [reply backend-nannyway/src/main/java/de/neuefische/nannyway/controller/ChildInCareController.java]: yeah.. ok ;)

File: backend-nannyway/src/main/java/de/neuefische/nannyway/model/AddChildInCareDto.java (hunk 2 above)
Comment (Cajan19) [reply backend-nannyway/src/main/java/de/neuefische/nannyway/model/AddChildInCareDto.java:16]: Done... what a struggle 😭

File: backend-nannyway/src/test/java/de/neuefische/nannyway/controller/ChildInCareControllerTest.java (hunk 3 above)
Comment (Cajan19) [reply backend-nannyway/src/test/java/de/neuefische/nannyway/controller/ChildInCareControllerTest.java:27]: done

**Pull request merged**